app/src/main/AndroidManifest.xml

```diff
@@ -14,16 +14,22 @@
     <uses-permission android:name="${applicationId}.permission.C2D_MESSAGE" />
 
     <application
-        android:icon="@mipmap/ic_launcher"
+        android:icon="@drawable/home_logo"
         android:label="@string/app_name"
         android:name="com.xmartlabs.doapp.DoAppApplication"
-        android:theme="@style/AppTheme">
+        android:theme="@style/AppTheme"
+        >
 
         <activity
             android:name="com.xmartlabs.doapp.ui.MainActivity"
             android:screenOrientation="portrait"
             />
 
+        <meta-data
+            android:name="io.fabric.ApiKey"
+            android:value="fb54b0fcded49d3e82043de10c558d9ea0018076"
```

Comment:
This key is no longer necessary here and can be put in a `fabric.properties` file. This helps to separate assets from code.